api/utils/helpers.py

```diff
@@ -10,7 +10,7 @@
 
 
 # Function to generate the 8 character short url
-def generate_short_url(original_url: str, timestamp: float):
+def generate_short_url(original_url: str, timestamp: float, shortened_length: int = 8):
```

Comment:
I'm not immediately familiar with the Python int - is that signed? unsigned? What happens if I pass -1. That length wouldn't make sense. How would -1 be interpreted in the operation on line 22?

Reply:
My goal was to remove the "magic number" maybe it should be set as an environment variable or application constant.  It would help prevent negative numbers.